spec/requests/bragi/items_api_spec.rb

```diff
@@ -150,6 +150,36 @@ module Bragi
           expect(json['data'][5]['id']).to eq item3.id.to_s
           expect(json['data'][6]['id']).to eq item2.id.to_s
         end
+
+        it 'fetches by after_id' do
+          get '/items', params: { filter: { after_id: item6.id } }, headers: { 'Authorization' => 'authorized_user' }
+
+          expect(response).to have_http_status(200)
+          json = JSON.parse response.body
+          expect(json['data'].size).to eq 3
+          json['data'].each do |item|
+            expect(item['attributes']['status']).to be_in(%w[unplayed playing])
+          end
+        end
+
+        it 'fetches by after_id and status' do
+          get '/items', params: { filter: { after_id: item6.id, status: 'unplayed' } }, headers: { 'Authorization' => 'authorized_user' }
+
+          expect(response).to have_http_status(200)
+          json = JSON.parse response.body
+          expect(json['data'].size).to eq 2
+          json['data'].each do |item|
+            expect(item['attributes']['status']).to eq 'unplayed'
```

Comment:
Same feedback as above: it'd be great if this checked that the contents of `json['data']` are the items that you expect.